src/images/reset.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="70" height="70" viewBox="0 0 70 70" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g filter="url(#filter0_d)">
+<circle cx="30" cy="31" r="25" fill="white"/>
+</g>
+<path d="M18.1182 27.3562L20.6048 25.8621C21.3504 25.4141 22.318 25.6554 22.7659 26.4009C23.2139 27.1465 22.9726 28.114 22.2271 28.562L15.3546 32.6914L11.2252 25.8189C10.7772 25.0734 11.0184 24.1058 11.764 23.6578C12.5096 23.2099 13.4771 23.4511 13.9251 24.1966L15.1592 26.2505C16.4767 21.831 19.6914 18.1254 24.0966 16.2554C32.1012 12.8577 41.3452 16.594 44.7437 24.6004C48.1422 32.6068 44.4085 41.8518 36.4039 45.2496C30.6239 47.703 24.0173 46.4758 19.5041 42.2659C18.868 41.6726 18.8334 40.676 19.4267 40.04C20.02 39.4039 21.0165 39.3693 21.6526 39.9625C25.2641 43.3313 30.5488 44.3131 35.1732 42.3501C41.5763 39.6322 44.5631 32.2364 41.8443 25.8311C39.1254 19.4258 31.7304 16.4369 25.3274 19.1548C21.7347 20.6798 19.1328 23.7292 18.1182 27.3562V27.3562Z" fill="#4A4453"/>
+<defs>
+<filter id="filter0_d" x="0" y="0" width="70" height="70" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
```

Comment:
Well actually you can just use `justKidding-haveNoClueAboutSvg`

Reply:
🤣